packages/suite/src/support/messages.ts

```diff
@@ -6683,4 +6683,16 @@ export default defineMessages({
         defaultMessage:
             'Your Trezor is running unofficial firmware. Please contact help@trezor.io immediately.',
     },
+    TR_TO_SATOSHIS: {
+        id: 'TR_TO_SATOSHIS',
+        defaultMessage: 'To sats',
```

Comment:
```suggestion
        defaultMessage: 'To Satoshis',
```

Reply:
I wanted to use the same unit name as everywhere, also _Satoshis to sats_ is the same as _Bitcoin to BTC_, right?

Reply:
You are right, to be consistent it should be either `To BTC`/`To sat` or `To Bitcoin`/`To Satoshis`
Bitcoin units: Bitcoin
<img width="139" alt="image" src="https://user-images.githubusercontent.com/3729633/180016942-baef6db8-4889-414b-8eb3-e46ad629e814.png">

Bitcoin units: Satoshis
<img width="158" alt="image" src="https://user-images.githubusercontent.com/3729633/180016955-0969ad76-18c0-497a-987b-aed8a5c021f7.png">

But maybe there could be some better label in the tooltip, @b-irving? Something like `Switch units to Satoshis/Bitcoin`?

Reply:
Will keep `To BTC/To sat` for consistency for now

Reply:
00f8d21751a7d216b265e462eca19f4118472013